scripts/azcopy.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
```

Comment:
If the script is meant to be used as a module and standalone, could you please make it executable (`chmod +x`)?